Cargo.toml

```diff
@@ -10,3 +10,5 @@ rpassword = "^2.0.0"
 simple_logger = "^0.5.0"
 tokio-core = "^0.1.17"
 web3 = "^0.4.0"
+rlp = "^0.3.0"
+sha3 = "^0.8.1"
```

Comment:
`web3` provides SHA3 functionality. Do we really need this extra dependency? Did you check that this is the most popular crate for hashing?

Reply:
web3 provides 'sha3' but it's an RPC call. I don't think it's a good idea to make an RPC call for `sha3`.

Reply:
OK. Did you check that this is the most popular crate for hashing?

Reply:
I had two options to choose from many available crates 
1.  [tiny-keccak](https://crates.io/crates/tiny-keccak)
2.  [sha3](https://crates.io/crates/sha3)
Both look promising but `sha3` was easy to use. 

But I just checked other projects like parity, they have used `tiny-keccak`. I will probably change this.